dlib/simd/simd8f.h

```diff
@@ -3,11 +3,9 @@
 #ifndef DLIB_sIMD8F_Hh_
 #define DLIB_sIMD8F_Hh_
 
-#include "simd_check.h"
```

Comment:
I agree.  But I suspect there might be user code out there that already includes the original files.  So it's best to avoid possibly disruption.